jerry-libc/jerry-libc.c

```diff
@@ -263,3 +263,21 @@ srand (unsigned int seed) /**< new seed */
   libc_random_gen_state[2] =
   libc_random_gen_state[3] = seed;
 } /* srand */
+
+/**
+ * Convert an integer to a string using base and write the result to buffer.
+ */
+char *
+itoa (int value, char *buffer, int base)
```

Comment:
This implementation is not quite complete. It doesn't add the `-` sign for negative numbers and only accepts bases <= 16.